src/models/soldier.js

```diff
@@ -0,0 +1,35 @@
+const rankMap = {
+	0: "private",
+	1: "corporal",
+	2: "sergeant",
+	3: "lieutenant",
+	4: "captain",
+	5: "major",
+	6: "colonel",
+};
+
+function getSoldierRankAndName(rankName, rankValue) {
```

Comment:
it returns the soldier's rank, so:
```suggestion
function getSoldierRank(rankName, rankValue) {
```